hooks/pyproject-without-special-deps.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python
-# Patch out special dependencies (git and path) from a pyproject.json file
+# Patch out special dependencies (git and path) from a pyproject.toml file
 
 import argparse
-import json
 import sys
 
+import tomlkit
+
 
 def main(input, output, fields_to_remove):
-    data = json.load(input)
+    data = tomlkit.loads(input.read())
```

Comment:
Would `tomlkit.load(input)` work?

Reply:
I think it will, let me check. I was trying to use a non-overlayed Python before, which then could be Python 2.7. In that case we got an ancient version of `tomlkit` that didn't have those functions yet.

Reply:
But since I'm using an non-overlayed Python now, it should just work.

Reply:
It doesn't :(

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pip-build-hook
Using pipBuildPhase
Using pipShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/h7bc0m662rcbci6x107allj8wkkkwa15-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tox.ini
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
Removing path dependencies
Traceback (most recent call last):
  File "/nix/store/6nqqz004095n2b5h55czcqvqidqngnv8-pyproject-without-special-deps.py", line 53, in <module>
    main(args.input, args.output, args.fields_to_remove)
  File "/nix/store/6nqqz004095n2b5h55czcqvqidqngnv8-pyproject-without-special-deps.py", line 11, in main
    data = tomlkit.load(input)
AttributeError: module 'tomlkit' has no attribute 'load'
```